internal-packages/next-dev/src/index.ts

```diff
@@ -207,3 +215,94 @@ function shouldSetupContinue(): boolean {
 	)['AsyncLocalStorage'];
 	return !!AsyncLocalStorage;
 }
+
+type MiniflareBindingOptions = Pick<
+	WorkerOptions,
+	| 'bindings'
+	| 'kvNamespaces'
+	| 'durableObjects'
+	| 'r2Buckets'
+	| 'd1Databases'
+	| 'serviceBindings'
+>;
+
+async function getMiniflareBindingOptions(
```

Comment:
@RamIdeas this is the mapping function 🙂 

but anyways this function is just temporary, as soon as `getBidingsProxy` is released, most of the `next-dev` code (including this function) will just go away and replaced with a call to the utility 🙂

(there will not be any user facing change as the API should be the same 🙂)